Dockerfile

```diff
@@ -43,11 +42,12 @@ RUN helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx && \
 # pulling `terra-app-setup` locally and add cert files to the chart.
 # Leonardo will install the chart from local version.
 # We are also cacheing charts so they are not downloaded with every helm-install
+# If you are changing any versions here, update it in the reference.conf
 RUN pushd /leonardo && \
-    helm pull terra-app-setup-charts/terra-app-setup --untar && \
-    helm pull galaxy/galaxykubeman --untar && \
-    helm pull terra/terra-app --untar && \
-    helm pull ingress-nginx/ingress-nginx --untar && \
+    helm pull terra-app-setup-charts/terra-app-setup --version 0.0.2 --untar && \
```

Comment:
wonder if it's better to keep these versions as `ENV` on top of the file

Reply:
I don't feel strongly either way. Could make it easier to find, but I personally search for `helm pull` when looking for these versions

Reply:
+1 on keeping the versions as ENV on top of the file for better visibility